src/include/network/lan/udev.rb

```diff
@@ -45,6 +45,23 @@ def update_udev_rule_key(rule, key, value)
       rule
     end
 
+    # Returns a value of the particular key in the rule
+    #
+    # @return [string] value corresponding to the key or empty string
+    def udev_key_value(rule, key)
+      raise ArgumentError if rule.nil?
```

Comment:
it is good practice to print also which argument have which problem so something like `raise ArgumentError, "rule must not be nil" if rule.nil?`